src/main/java/seedu/address/model/person/Mark.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the mark status in the address book.
+ * Guarantees: mutable.
+ */
+public class Mark {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Mark Status can only be true or false!";
```

Comment:
More minor nits to imply variable type. Probably not needed, but if you want!
```suggestion
    public static final String MESSAGE_CONSTRAINTS =
            "Mark Status can only be a boolean of true or false!";
```